config/overlays/release/envoy_proxy_patch.yaml

```diff
@@ -0,0 +1,43 @@
+# To customize EnvoyProxy
+# https://gateway.envoyproxy.io/docs/tasks/operations/customize-envoyproxy/
+apiVersion: gateway.envoyproxy.io/v1alpha1
+kind: EnvoyProxy
+metadata:
+  name: custom-proxy-config
+  namespace: aibrix-system
+spec:
+  provider:
+    type: Kubernetes
+    kubernetes:
+      envoyDeployment:
+        replicas: 1
```

Comment:
```
patch:
          type: StrategicMerge
```

is this deprecated in latest version?

Reply:
It is not deprecated. For our purpose of overriding replica or resources, we do not need strategic merge as simple direct constructs are [available](https://github.com/envoyproxy/gateway/blob/6a15dc769f647905498a0322f425e7eabef04e09/api/v1alpha1/shared_types.go#L63).